### ISSUE TITLE
verdi export create -c does not work

### ISSUE THREAD

**Issue by broeder-j** [aiidateam/aiida_core#1033]
```verdi export create --computers 2 -a tar.gz "computers_"$(date +%Y%m%d)```
```<Computer: iff003 (iff003), pk: 2>```

fails with 
```
 File "/Users/broeder/aiida/github/aiida_core/aiida/orm/importexport.py", line 1699, in export_tree
    raise ValueError("I was given {}, which is not a DbNode or DbGroup instance".format(entry))
ValueError: I was given iff003 (iff003), which is not a DbNode or DbGroup instance
```
My guess it that it apparently was able to load/get the computer, but Computer fails then in the more general functions, which do not expect that type.

Also
```verdi export create --computers 1 2 3 4 -a tar.gz "computers_"$(date +%Y%m%d)```

Fails with:
```
Usage: verdi export create [OPTIONS] OUTFILE
Error: Got unexpected extra arguments (3 4 computers_20180109)
```

Therefore the input parsing is not working for several pks, in the computer case.

**Comment by broeder-j** [comment]
update: computer exports work fine with other nodes, like codes. Also the import. 
Therefore, this issue is not critical or blocking and prob a bug in the --computer export case. But so far I do not see in the code why it should run into this at all. 

If you need to export or import only computers, as a work around one can go back to version 0.9. there it works.

**Comment by szoupanos** [comment]
Yes, maybe we missed the case that someone wants to export only computers and no other nodes. We will have a look at that.

**Comment by sphuber** [comment]
Fixed in #1448
